src/front/app/features/challenge/challenge.actions.js

```diff
@@ -26,15 +27,18 @@ export function fetchChallenge(id) {
 
 export function watchChallenge(challengeId) {
     return dispatch => api(`/api/my/wish-list/${challengeId}`, { method: 'PUT' })
-        .then(() => {
-            dispatch({ type: ADDED_TO_WATCHLIST, challengeId });
-            // eslint-disable-next-line no-alert
-            window.alert('The challenge has been added to your watch list.');
-        })
+        .then(() => dispatch({ type: ADDED_TO_WATCHLIST, challengeId }))
         // eslint-disable-next-line no-alert
         .catch(alert.bind(null, 'Unable to add the challenge =('));
```

Comment:
I suggest none for now. It should be done commonly. And this logic will only obstruct implementing of common flow.